uwblueprint/starter-code-v2
# Front-end Create and Update

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/REST-client-for-Entity-endpoints-57d23ba28155464e8d63a19d977f119e)

Tentative, still fixing some styling issues


## Implementation description
*  Update and Create components with functions similar to Auth


## Steps to test
1. Click Create Button and enter your desired fields
2. Click the submit button
3. Make a get request to localhost:5000/entities and verify that entity was created, grab that ID
4.  Go back to main page and click Update Button, enter desired fields
5. Click submit button
6. Make a get request to localhost:5000/entities and verify that entity was updated



## What should reviewers focus on?
* CrudAPIClient
* CreatePage
* UpdatePage
* Create
* Update


## Checklist
- [* ] I have run the appropriate linter(s)
- [* ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
